agent/agent.go

```diff
@@ -8,30 +8,33 @@ import (
 	"sync"
 	"time"
 
+	"github.com/hashicorp/consul-template/manager"
 	consul "github.com/hashicorp/consul/api"
 	"github.com/pkg/errors"
-	"github.com/hashicorp/consul-template/manager"
+	"github.com/trento-project/trento/agent/discover"
 )
 
 type Agent struct {
 	cfg              Config
 	check            Checker
+	discovers        []discover.Discovery
```

Comment:
little typo
```suggestion
	discoveries        []discover.Discovery
```

Reply:
changed that (together with the go fmt fallout)